### ISSUE TITLE
Ability to enter package version

### ISSUE THREAD

**Issue by benmccann** [cheapsteak/npmcharts.com#189]
Thanks so much for providing this service! I'd love to be able to use it to track adoption of Svelte 3 vs Svelte 4. It might be a nice feature to be able to enter `svelte@^4` or something along those lines

**Comment by benmccann** [comment]
Oh, hmm. Yeah, I guess that might kill the idea then if there's not a way to get the data

**Comment by cheapsteak** [comment]
Might still be interesting though to show which are the prevalent versions as additional metadata, even if it's not chart-able over time 💭 

But yeah wish there were an easy way to get the data

**Comment by benmccann** [comment]
Yeah, that would still be valuable. If I go to https://www.npmjs.com/package/svelte?activeTab=versions it's a bit hard to tell how many people are on v3 vs v4 because I have to add up all the minor versions to get the total
